.github/workflows/e2e-manual-icad.yaml

```diff
@@ -25,8 +25,8 @@ on:
         default: master
         options:
           - master
-          - v0.4.2
-          - v0.3.5
+          - v0.4.3
+          - v0.3.6
```

Comment:
I'll create these tags in interchain-accounts-demo with v5.2.0 and v6.1.0.

Reply:
https://github.com/cosmos/interchain-accounts-demo/releases/tag/v0.3.6 and https://github.com/cosmos/interchain-accounts-demo/releases/tag/v0.4.3